packages/editor/api-report.md

```diff
@@ -433,6 +439,8 @@ export class Editor extends EventEmitter<TLEventMap> {
     readonly isSafari: boolean;
     isSelected(id: TLShapeId): boolean;
     isShapeInPage(shape: TLShape, pageId?: TLPageId): boolean;
+    isShapeOfType<Shape extends TLUnknownShape>(shape: TLUnknownShape, definition: TLShapeInfo<Shape>): shape is Shape;
```

Comment:
This is fine but we can't rely on this inside of Editor, as we'll be moving our shape definitions out of the editor.

Reply:
Yep, that's intentional - the idea here is we have the string version for us to use internally in @tldraw/editor, but the nicer type-safe one for external API consumers/packages that have access to shape defs